Laba2SCCS/src/CarInspection/Controllers/DriverController.cs

```diff
@@ -0,0 +1,198 @@
+﻿using BusinessLayer.Entities;
+using BusinessLayer.Infrastructe;
+using BusinessLayer.Interfaces;
+using EasyConsole;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CarInspection.Controllers
+{
+    public class DriverController
+    {
+        private readonly IService<Driver> _driverService;
+
+        public DriverController(IService<Driver> driverService)
+        {
+            _driverService = driverService;
+        }
+
+        public void Start()
```

Comment:
I guess you can create an interface IController that will contain Start method and reuse this interface for consistency.